docs/getting-started.md

```diff
@@ -383,77 +284,3 @@ Points to check to confirm the API socket is running and accessible:
   - If you have socat available, try `socat - UNIX-CONNECT:/tmp/firecracker.socket`
     This will throw an explicit error if the socket is inaccessible, or it will pause
     and wait for input to continue.
-
-## Appendix A: Setting Up KVM Access
-
-Some Linux distributions use the `kvm` group to manage access to `/dev/kvm`,
-while others rely on access control lists. If you have the ACL package for your
-distro installed, you can grant your user access via:
-
-```bash
-sudo setfacl -m u:${USER}:rw /dev/kvm
-```
-
-Otherwise, if access is managed via the `kvm` group:
-
-```bash
-[ $(stat -c "%G" /dev/kvm) = kvm ] && sudo usermod -aG kvm ${USER} \
-&& echo "Access granted."
-```
-
-If none of the above works, you will need to either install the file
-system ACL package for your distro and use the `setfacl` command as above,
-or run Firecracker as `root` (via `sudo`).
-
-You can check if you have access to `/dev/kvm` with:
-
-```bash
-[ -r /dev/kvm ] && [ -w /dev/kvm ] && echo "OK" || echo "FAIL"
-```
-
-**Note:** If you've just added your user to the `kvm` group via `usermod`, don't
-forget to log out and then back in, so this change takes effect.
-
-## Appendix B: Setting Up Docker
```

Comment:
We might need to discuss if we should remove this section.

Reply:
I don't think there is value in documenting the docker install process here.